a10_ansible/axapi_http.py

```diff
@@ -300,21 +300,36 @@ def delete(self, url, params={}, **kwargs):
 
     def activate_partition(self, partition):
         url = "/axapi/v3/active-partition"
-        payload = { 
+        payload = {
             "curr_part_name": partition["name"],
             "shared": partition["shared"]
         }
         try:
             self.post(url, {"active-partition": payload})
         except Exception as ex:
-            raise Exception("Could not activate due to: {0}".format(ex)) 
+            raise Exception("Could not activate due to: {0}".format(ex))
```

Comment:
Please change this to can an ACOSException found here: https://github.com/a10networks/a10-ansible/blob/master/a10_ansible/errors.py#L16